test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -18,6 +21,61 @@ Class WebListener
 
 [WebListener]$WebListener
 
+function New-ClientCertificate
+{
+    param([string]$CertificatePath)
+
+    $password = ConvertTo-SecureString -Force -AsPlainText 'password'
```

Comment:
We shouldn't have a hardcoded password.  Looks like the `GeneratePassword()` method isn't available in corefx.  I think it's fine to generate a random alphanumeric password.